tools/hitmap/hitmap.xml

```diff
@@ -0,0 +1,297 @@
+
+<tool id="hitmap" name="HiT-MaP" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" license="MIT">
+    <description>High-resolution Maldi-imaging Proteomics Analysis</description>
+
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+
+    <edam_topics>
+        <edam_topic>topic_0121</edam_topic>
+    </edam_topics>
+
+    <edam_operations>
+        <edam_operation>operation_3630</edam_operation>
+    </edam_operations>
+
+    <requirements>
+        <container type="docker">mashuoa/hitmap@sha256:97f2f853362b56be6c2b0fb1507e0288667e7fb6ad0eda3ff1d761dd72dfaad0</container>   
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+        mkdir expdata &&
+        cp '${file_inputs.imzml_file}' expdata/sample.imzML &&
+        cp '${file_inputs.ibd_file}' expdata/sample.ibd &&
+        cp '${file_inputs.fasta_db}' expdata/database.fasta &&
+        Rscript '${hitmap_script}' &&
+        python3 $__tool_directory__/hitmap_gen_html.py expdata &&
```

Comment:
single quote for the path

Reply:
do you mean $__tool_directory__/hitmap_gen_html.py -> '$__tool_directory__/hitmap_gen_html.py'?

Reply:
yes :) the reason is that you might have "spaces" in your tool_directory path .. in this case the tool breaks.

Reply:
haha oh yeah that's a great point. Can happen! 
Added to the next commit :)